src/rust/e2e-tests/src/test_utils/context.rs

```diff
@@ -255,11 +255,24 @@ impl E2eTestContext {
         Ok(())
     }
 
+    pub async fn setup_suspicious_svchost_analyzer(
+        &mut self,
+        tenant_id: uuid::Uuid,
+        test_name: &str,
+    ) -> eyre::Result<uuid::Uuid> {
+        let analyzer_artifact = test_fixtures::get_suspicious_svchost_analyzer()?;
+        let analyzer_plugin_id = self
+            .create_analyzer(tenant_id, test_name.to_owned(), analyzer_artifact)
```

Comment:
This especially doesn't matter because it's a test fn, but just fwiw I would say that if you're going to require a String internally you should take a String in your API, *or* `impl Into<String>`, which would let callers pass in &str or String.

Purely style, pls do not feel like you have to fix this

Reply:
cool 2 know